tests/Umbraco.AuthorizedServices.Tests/Services/AuthorizedServiceAuthorizerTests.cs

```diff
@@ -14,7 +14,8 @@ internal class AuthorizedServiceAuthorizerTests : AuthorizedServiceTestsBase
     [SetUp]
     public void SetUp()
     {
-        TokenStorageMock = new Mock<ITokenStorage>();
+        TokenStorageMock = new Mock<ITokenStorage<Token>>();
```

Comment:
As well as updating the existing tests so they still pass as you have here, don't forget to also create unit tests for the new methods that you have introduced.

Reply:
Revisited existing tests and corrected. Additional test cases will follow with a different commit.